server.js

```diff
@@ -7,19 +8,20 @@ app.use(express.static(path.join(__dirname, '/public')));
 app.use(bodyParser.urlencoded({ extended: false }))
 app.use(bodyParser.json());
 
-app.use('/', express.static(path.join(__dirname, 'public'));
+app.use('/', express.static(path.join(__dirname, 'public')))
```

Comment:
Since the rest of the app is using trailing semi-colons, I recommend continuing that style.